### ISSUE TITLE
Highlight particular days

### ISSUE THREAD

**Issue by fabiodcorreia** [BelkaLab/react-yearly-calendar#4]
Hi,

It's possible to bind a list of dates (moment) to be highlighted on the calendar like event, for example officials holidays or birthdays? A limitation fo 1 event per days should work for my use case.

Best Regards,
Fábio Correia

**Comment by GiovanniFrigo** [comment]
Hi, 
currently we have no such feature, but it is planned for the upcoming release 1.1. 

I was thinking of adding a prop that tells the calendar to add custom css classes to single days or range of days, something like this:

```
customClasses: {
    "holidays": [ ..array of moment.js days ],
    "birthdays":  [ ..array of moment.js days ],
    "unselectable-days": [ array of moment.js days ],
    ..
}
```

You would then be able to style, for example, _.holidays_ and _.birthdays_ with custom colors and _.unselectable-days_ with `pointer-events: none`
Would that work for you or were you thinking of something else?

**Comment by fabiodcorreia** [comment]
My use case is to hight light the the days with different colors, for example week-ends on gray, holidays on orange, daysX on red, daysY on blue and so one.

So I think this way we can send a object with N classes and each classes with an array of dates to highlight. it's very flexible this way and should work for my case.

**Comment by GiovanniFrigo** [comment]
This feature is now available in version 1.1.0 🎉
